src/govuk/components/checkboxes/checkboxes.yaml

```diff
@@ -279,10 +279,21 @@ examples:
     items:
       - value: british
         text: British
+        hint:
```

Comment:
I think this would be better served through adding new tests, as this fixture is meant to test checkboxes with 'all fieldset attributes' and these changes don't really fit that description.

Reply:
I've refactored the tests so the fixtures are specific for each test.